rllib/agents/impala/impala.py

```diff
@@ -194,6 +194,8 @@ def __call__(self, item):
             metrics = _get_shared_metrics()
             metrics.counters["num_weight_broadcasts"] += 1
         actor.set_weights.remote(self.weights, _get_global_vars())
+        # Also update global vars of the local worker.
+        self.workers.local_worker().set_global_vars(_get_global_vars())
```

Comment:
Does the LR not update without this line of code?

Reply:
I think this was the root-cause of this issue: That the local worker's on_global_var_update was never called. The remote workers' lr was always fine and updated correctly.

Reply:
So, yes, w/o this line, lr would not get updated on the driver.

Reply:
Got it! This is the core line of the LR PR :).